app/graphql/types/input/titles_list.rb

```diff
@@ -1,4 +1,6 @@
 class Types::Input::TitlesList < Types::Input::Base
+  graphql_name 'TitlesListInput'
```

Comment:
Wouldn't these both autogenerate properly now, since this PR fixes the default_graphql_name?

Reply:
You are entirely right! I was thinking of it differently (needing to be nested).